CONTRIBUTING.md

```diff
@@ -20,6 +20,7 @@ information on using pull requests.
 Prerequisites:
 
 * You need to have Python installed.
+* You must have every version of Python that OpenCensus supports installed for testing (see https://pypi.org/project/opencensus/ for which versions are supported).
```

Comment:
"every version" -> "every major version" since we don't distinguish minor versions (e.g. 3.6.1 and 3.6.2).

Reply:
Good point. We don't want them installing every single release!